scripts/gulpfiles/package_tasks.js

```diff
@@ -126,8 +126,8 @@ function packageBlockly() {
  */
 function packageBlocks() {
   return gulp.src('scripts/package/blocks.js')
-    .pipe(packageUMD('Blockly.Blocks', [{
-        name: 'Blockly',
+    .pipe(packageUMD('BlocklyBlocks', [{
```

Comment:
Why did you keep the `scripts/package/thing.js` files, while emptying them of code, instead of just cutting that step off the pipeline entirely? Is there a benefit to starting the pipeline with an actual file, or is it just serving to inject the license and comment about the wrapper?

Reply:
Mainly to try to not break anything, TBH, but yes: the documentation in the resulting files in `dist/` is helpful (and this improved documentation might have helped me considerably when I started trying to understand this mess some months ago).

I eventually think most of this should go away:

- Why not just copy `build/blockly_compressed.js` to `dist/blocklyjs` instead of having a separate wrapper module?
- Even where a wrapper module is needed, these ones are all so simple that having a build pipeline with a separate build function to put a UMD wrapper around each of module is absurdly heavyweight.  Surely we could just put pre-UMD-wrapped static files in `scripts/package` and just copy them into `dist/` unmodified?

But in the mean time I figured that feeding an effectively-empty file to the existing pipeline was the safest thing to do for the time being.